docker-compose.yml

```
@@ -168,4 +168,4 @@ networks:
 
 volumes:
     techeerism-db:
-    meilisearch-data:
+    meilisearch-data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)